### PR TITLE
RHIDP-2647 : chore: updates from privacy-legal team RHIDP-58

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: updates from privacy-legal team RHIDP-58

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: assemblies/assembly-rhdh-telemetry.adoc
Comment (abrennan89) [inline assemblies/assembly-rhdh-telemetry.adoc:19]: You could probably just remove this line, since it's pretty much repeating the next line. I'd just make this one combined bullet list.

File: assemblies/assembly-rhdh-telemetry.adoc
Comment (abrennan89) [inline assemblies/assembly-rhdh-telemetry.adoc:11]:
```suggestion
{company-name} analyzes collected data to understand your interactions with the application, and to improve offerings and your experience with {product}.
```
suggested editing

**Review by abrennan89 — APPROVED** [review]

**Review by linfraze — DISMISSED** [review]
A couple suggested changes. Otherwise, lgtm, Will re-approve, if needed.

File: assemblies/assembly-rhdh-telemetry.adoc
Comment (abrennan89) [reply assemblies/assembly-rhdh-telemetry.adoc:19]: Thought about suggesting this but didn't want to hold up the PR 🙂

**Comment by briandooley** [comment]
I've created [RHIDP-2647](https://issues.redhat.com/browse/RHIDP-2647) to process this.

**Comment by shipsing** [comment]
The content changes look fine, @nickboldt. Thanks

**Review by shipsing — APPROVED** [review]

**Review by zdrapela — APPROVED** [review]

**Comment by nickboldt** [comment]
From @shipsing : " I think we can proceed with merge because the usual review process is complete and we have notified legal as well in that mail. "

Therefore merging.

**Pull request merged**
